docs/eventing/broker/kafka-broker/README.md

```diff
@@ -59,6 +59,8 @@ metadata:
   annotations:
     # case-sensitive
     eventing.knative.dev/broker.class: Kafka
+    # Optional annotation to point to an externally managed kafka topic:
+    # kafka.eventing.knative.dev/external.topic: <topic-name>
```

Comment:
I'd put this example in the `Using externally managed topic` section since:
- this should be the default for most users
- in that section we can explain what are the implications of using an external topic (topic lifecycle, validity, restrictions, ACLs)

Reply:
Interesting, I had it on an extra separate section, I ended up moved it here, b/c I was expecting a comment that I should move to the large one to not duplicate content :joy: 

I move up, as I don't really care where we document.

Regarding ACLs, I am for sure not explaining all the ACL, I will add a warning / note with some implications